ui/src/main/js/application/MainPage.js

```diff
@@ -144,28 +143,14 @@ const MainPage = ({
                 urlName={EMAIL_INFO.url}
                 descriptor={globalDescriptorMap[EMAIL_INFO.key]}
                 render={(readOnly, showTest, showSave, showDelete) => (
-                    <BetaPage>
-                        <CurrentComponent>
-                            <EmailGlobalConfiguration
-                                csrfToken={csrfToken}
-                                errorHandler={errorHandler}
-                                readonly={readOnly}
-                                displayTest={showTest}
-                                displaySave={showSave}
-                                displayDelete={showDelete}
-                            />
-                        </CurrentComponent>
-                        <BetaComponent>
-                            <EmailGlobalConfigurationStandalone
-                                csrfToken={csrfToken}
-                                errorHandler={errorHandler}
-                                readonly={readOnly}
-                                displayTest={showTest}
-                                displaySave={showSave}
-                                displayDelete={showDelete}
-                            />
-                        </BetaComponent>
-                    </BetaPage>
+                    <EmailGlobalConfiguration
+                        csrfToken={csrfToken}
+                        errorHandler={errorHandler}
+                        readonly={readOnly}
+                        displayTest={showTest}
+                        displaySave={showSave}
+                        displayDelete={showDelete}
+                    />
```

Comment:
Maybe we should put the new page as current and the old one under beta, just in case we run into issues with any of the functionality?

Reply:
I will add the old configuration as EmailGlobalConfigurationOld and set it up as a beta page